kythe/go/serving/explore/explore.go

```diff
@@ -49,17 +57,108 @@ func (t *Tables) TypeHierarchy(ctx context.Context, req *epb.TypeHierarchyReques
 	return nil, nil
 }
 
-// Callers returns the (recursive) callers of a specified function, as a directed graph.
-// TODO: not yet implemented
+// Callers returns the callers of a specified function, as a directed graph.
 func (t *Tables) Callers(ctx context.Context, req *epb.CallersRequest) (*epb.CallersReply, error) {
-	return nil, nil
+	tickets := req.Tickets
+	if len(tickets) == 0 {
+		return nil, fmt.Errorf("missing input tickets: %v", &req)
```

Comment:
s/%req/req/

Reply:
I'm going to assume you meant "s/&req/req" since there aren't any "%req" here.  Yeah, I'm not sure how I ended up with `&req`, thanks.